browser/src/VariantPage/VariantPage.tsx

```diff
@@ -146,8 +146,10 @@ export type Population = {
   id: PopulationId
   ac: number
   an: number
-  ac_hemi: number | null
-  ac_hom: number
+  ac_hemi?: number | null
```

Comment:
Do we in fact want the `?` here? I.E. can these really be `number | null | undefined`?

Reply:
Ah good point, I rather short-sightedly added the `?` since it was getting assigned from another type that can be `undefined` in a particular circumstance, though on second thought in that circumstance (joint data exists) we know the value won't be undefined -- good catch.

Will address in a fixup commit.